src/yadism/coefficient_functions/light/nlo/g1.py

```diff
@@ -0,0 +1,23 @@
+import numba as nb
+import numpy as np
+from eko.constants import TR
+
+from . import f2, f3
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def ns_reg(z, args):
+    return f3.ns_reg(z, args)
+
+
+# The coefficients are the same as
+# F2LIGHT_NS
+ns_delta = f2.ns_delta
+ns_omx = f2.ns_omx
+ns_logomx = f2.ns_logomx
+
+
+@nb.njit("f8(f8,f8[:])", cache=True)
+def gluon_reg(z, args):
+    nf = args[0]
+    return 4.0 * nf * TR * ((2.0 * z - 1.0) * np.log((1.0 - z) / z) - 4.0 * z + 3.0)
```

Comment:
I'm not sure about the origin of this `4.0` (the one in front of `nf`), but sometimes, to make their role more explicit, we named these factors after their role (i.e. `as_norm`, in those cases - and being NLO it is a likely candidate even fro this one).

Reply:
I'd leave it this way. This has been copied from `apfel` and follows exactly the equation given in the 
original [paper](https://reader.elsevier.com/reader/sd/pii/055032139490538X?token=9FDADFCED77D43FABB6A22975BB951E40B72D9B7321ED3861C912B9AE144B37FB22C37E30DD888E12AE1203B1681BBF9&originRegion=eu-west-1&originCreation=20230517063035)